eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -64,6 +64,10 @@ stages:
         parameters: 
           ServiceDirectory: ${{ parameters.ServiceDirectory }}
           GoWorkspace: $(GO_WORKSPACE_PATH)
+          
+      - template: ../../../common/pipelines/templates/steps/verify-links.yml
```

Comment:
Generally a better idea to use a full path here as it is easier to see the exact path so I suggest you use `/eng/common/pipelines/templates/steps/verify-links.yml`